src/ArticleList.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+import ArticleCard from './ArticleCard';
+
+export const ArticleList = ({ articleList, setArticleList }) => {
+
+    useEffect(() => {
+        fetch(`https://nc-news-project-9qpu.onrender.com/api/articles`)
+            .then((response) => {
+                return response.json();
+            })
```

Comment:
Rather than using fetch, see if you can refactor this to use axios and extract it to a separate api.js file 👍